sdks/python/apache_beam/examples/kafkataxi/README.md

```diff
@@ -185,5 +185,7 @@ python -m apache_beam.examples.kafkataxi.kafka_taxi \
   --job_name $JOB_NAME \
   --bootstrap_servers $BOOTSTRAP_SERVER \
   --sdk_harness_container_image_overrides ".*java.*,${DOCKER_ROOT}/beam_java8_sdk:latest" \
-  --experiments=use_runner_v2
+  --experiments=use_runner_v2 \
```

Comment:
I don't think extra options are needed. Just 'use_runner_v2' should be adequate for Dataflow on runner v2.

cc: @angoenka

Reply:
(note that options used for Google internal testing framework may be different. This example is for external usage).

Reply:
From my experience, using `use_runner_v2` only did give me java production runner instead of runner v2. We should revisit how these experiments get populated.

Reply:
cc: @ananvay 
This is surprising. If this is the case something might be broken at HEAD since we ask users to only specify use_runner_v2 to use the C++ worker: https://cloud.google.com/dataflow/docs/guides/deploying-a-pipeline#dataflow-runner-v2

Reply:
Not that I'm aware of -- this should still work fine. @angoenka FYI.

Reply:
Double checked it with running the cmd again. It seems like I was wrong and am sending out the revert PR: https://github.com/apache/beam/pull/13446

Reply:
Ack, thank you for double-checking Boyuan!

Reply:
Thanks for checking.